media/css/firefox/channel.scss

```diff
@@ -83,11 +83,17 @@ $image-path: '/media/protocol/img';
 // styles from developer/includes/newsletter.scss
 
 .t-newsletter {
+    background: #08204e url(/media/img/firefox/developer/newsletter-bg.5cb53dbe3960.svg) center bottom no-repeat;
```

Comment:
It looks like tests are failing due to this url helper's string not being wrapped in single quotes. 
`url('/media/img/firefox/developer/newsletter-bg.5cb53dbe3960.svg')`

[Here are the docs on running bedrock tests.](https://bedrock.readthedocs.io/en/latest/install.html#run-the-tests)